### PR TITLE
ipcalc: add page

### PR DESCRIPTION
**Pull request by raylas**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/ipcalc.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# ipcalc
+
+> Perform simple operations/calculations on IP addresses and networks.
```
Comment (sbrl) [inline pages/linux/ipcalc.md]: Perhaps ....`operations and calculations`.... would read better?

File: pages/linux/ipcalc.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show information about an address/network:
```
Comment (sbrl) [inline pages/linux/ipcalc.md]: Perhaps ...`address or network:` would read better?

File: pages/linux/ipcalc.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`ipcalc {{ip_address}} {{subnet_mask}}`
+
+- Show information about an address/network in CIDR notation:
```
Comment (sbrl) [inline pages/linux/ipcalc.md]: Same here.

File: pages/linux/ipcalc.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`ipcalc {{ip_address}}/{{masked_bits}}`
+
+- Show broadcast address of an address/network:
```
Comment (sbrl) [inline pages/linux/ipcalc.md]: Same here. Also perhaps `Show the broadcast`.... would read better?

File: pages/linux/ipcalc.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`ipcalc -b {{ip_address}}/{{masked_bits}}`
+
+- Show network address of provided IP address and netmask:
```
Comment (sbrl) [inline pages/linux/ipcalc.md]: Perhaps `Show the network address of the provided`.... would read better?

File: pages/linux/ipcalc.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+`ipcalc -n {{ip_address}}/{{masked_bits}}`
+
+- Display geographic information of provided IP address:
```
Comment (sbrl) [inline pages/linux/ipcalc.md]: Perhaps `Display geographic information about a given IP address:` would read better?

**Comment by raylas** [comment]
Thanks for the feedback, @sbrl — see updated commit.

**Comment by raylas** [comment]
@agnivade As in remove the curly braces and insert an example value in the place of `{{ip_address}}`, etc.? Do you suggest this for each instance?

File: pages/linux/ipcalc.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# ipcalc
+
+> Perform simple operations and calculations on IP addresses and networks.
+
+- Show information about an address or network:
+
+`ipcalc {{1.2.3.4}} {{255.255.255.0}}`
```
Comment (agnivade) [inline pages/linux/ipcalc.md]: I am a bit unclear on this. The description says - "address or network". But the example contains a subnet mask. Should we also say - "address or network with a given subnet mask" ?

Comment (raylas) [reply pages/linux/ipcalc.md]: @agnivade I've added the suggested language. It definitely helped with clarification.

**Pull request merged**